whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -620,43 +626,35 @@ private Domain createDomain(final RpslObject rpslObject, final String requestUrl
         if (secureDNS.isDelegationSigned()) {
             domain.setSecureDNS(secureDNS);
         }
-
-        domain.getEntitySearchResults().addAll(createContactEntities(rpslObject, requestUrl));
+        mapContactEntitiesAndRedaction(domain, rpslObject, requestUrl);
         return domain;
     }
 
     private static VCard createVCard(final RpslObject rpslObject) {
+        return commonVcard(rpslObject).build();
```

Comment:
Can you revert this change as it was before ? if its calling the method directly no point in creating two methods?

Reply:
Done